Shared/Samples/Identify KML features/IdentifyKMLFeaturesView.swift

```diff
@@ -0,0 +1,153 @@
+    @State private var error: Error? {
+        didSet { isShowingErrorAlert = error != nil }
+    }
+    
+    var body: some View {
+        MapViewReader { mapViewProxy in
+            GeometryReader { geometry in
+                MapView(map: map)
+                    .callout(placement: $calloutPlacement.animation(.default.speed(2))) { _ in
+                        Text(calloutText)
+                            .frame(maxWidth: geometry.size.width / 2)
+                            .font(.callout)
+                            .padding(8)
+                    }
+                    .onSingleTapGesture { screenPoint, _ in
+                        Task {
+                            do {
+                                // Get the KML placemark for the screen point.
+                                if let placemark = try await kmlPlacemark(for: screenPoint, using: mapViewProxy) {
+                                    // Update the callout's text and placement.
+                                    try updateCalloutText(using: placemark)
+                                    if let location = mapViewProxy.location(fromScreenPoint: screenPoint) {
+                                        calloutPlacement = .location(location)
+                                    }
+                                } else {
+                                    // Dismiss the callout if a placemark was not found.
+                                    calloutPlacement = nil
+                                }
+                            } catch {
+                                self.error = error
+                            }
+                        }
+                    }
+                    .task {
+                        do {
+                            // Load a KML dataset from a URL.
+                            let dataset = KMLDataset(url: .forecastKML)
+                            try await dataset.load()
+                            
+                            // Create a KML layer with the dataset.
+                            forecastLayer = KMLLayer(dataset: dataset)
+                            
+                            // Add the layer to the map.
+                            map.addOperationalLayer(forecastLayer!)
+                        } catch {
+                            self.error = error
+                        }
+                    }
+            }
+        }
+        .alert(isPresented: $isShowingErrorAlert, presentingError: error)
+    }
+}
+
+private extension IdentifyKMLFeaturesView {
+    /// Identifies the placemark for a given point on the KML layer.
+    /// - Parameters:
+    ///   - screenPoint: The screen point corresponding to a placemark.
+    ///   - proxy: The map view proxy used identify the screen point.
+    /// - Returns: The first KML placemark in the identify result.
+    private func kmlPlacemark(for screenPoint: CGPoint, using proxy: MapViewProxy) async throws -> KMLPlacemark? {
```

Comment:
```suggestion
    func kmlPlacemark(for screenPoint: CGPoint, using proxy: MapViewProxy) async throws -> KMLPlacemark? {
```
Since this method is declared in a private extension it's redundant to specify the access level here.